<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTestGui2</class>
 <widget class="QDialog" name="QTestGui2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAgentes Test GUI</string>
  </property>
  <widget class="QGroupBox" name="groupBoxInformacoesGerais">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>371</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Escolha os Agentes:</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>227</width>
      <height>124</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutEscolhaOsAgentes">
     <item>
      <widget class="QCheckBox" name="checkBoxAgenteSimples">
       <property name="text">
        <string>Agentes Simples</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAgenteMemoriaAnterior">
       <property name="text">
        <string>Agente com Memória Anterior</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAgenteGuloso">
       <property name="text">
        <string>Agente Guloso</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAgenteInteligente">
       <property name="text">
        <string>Agente Inteligente</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxEscolhaOsAgentes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Informações Gerais:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>351</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutInformaoesGerais">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPorcentagemDeSujeira">
       <property name="text">
        <string>Porcentagem de Sujeira:(em %)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelRepeticoes">
       <property name="text">
        <string>Repetições:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxRepeticoes">
       <property name="minimum">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxPorcentagemDeSujeira">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAmbiente">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Ambiente:</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayoutAmbiente">
     <item row="0" column="0">
      <widget class="QLabel" name="labelLargura">
       <property name="text">
        <string>Largura:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBoxLargura">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelAltura">
       <property name="text">
        <string>Altura:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxAltura">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>390</y>
     <width>188</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutBotoes">
    <item>
     <widget class="QPushButton" name="pushButtonExecutar">
      <property name="text">
       <string>Executar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancelar</sender>
   <signal>clicked()</signal>
   <receiver>QTestGui2</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>406</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
